docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -3,116 +3,123 @@ title: "Restarting"
+(for example, "rolling restarts") and sometimes all the pods need to be deleted and then recreated as part of a downtime (for example, "full restarts").
 
-The following types of server restarts are supported in Oracle WebLogic Server in Kubernetes:
+The following types of server restarts are supported by the operator:
 
 * Rolling restarts - a coordinated and controlled shut down of all of the servers in a domain or cluster while ensuring that service to the end user is not interrupted.
 
-   * Operator initiated - where the WebLogic Server Kubernetes Operator can detect some types of changes and will automatically initiate rolling restarts of server pods in a WebLogic domain.
+   * Operator initiated - where the WebLogic Server Kubernetes Operator can detect some types of changes and will automatically initiate rolling restarts of pods in a domain or cluster.
 
    * Manually initiated - required when certain changes in the Oracle WebLogic Server in Kubernetes environment cannot be detected by the operator, so a rolling restart must be manually initiated.
 
 * Full domain restarts - the Administration Server and all the Managed Servers in a domain are shutdown, impacting service availability to the end user, and then restarted.  Unlike a rolling restart, the operator cannot detect and initiate a full domain restart; it must always be manually initiated.
 
-For detailed information on how to restart servers in a Oracle WebLogic Server in Kubernetes environment, see [Starting, stopping, and restarting servers]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md" >}}).
+For detailed information on how to restart servers using the operator, see [Starting, stopping, and restarting servers]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md" >}}).
 
 #### Common restart scenarios
 
 This document describes what actions you need to take to properly restart your servers for a number of common scenarios:
 
-* Modifying the WebLogic configuration
-* Changing the custom domain configuration overrides (also called situational configuration) for Domain in PV and Domain in Image domains
+* Modifying the WebLogic domain configuration
+* Changing the domain configuration overrides (also called situational configuration) for Domain in PV and Domain in Image domains
 * Changing the model files for Model in Image domains
 * Changing the WebLogic Server credentials (the user name and password)
-* Changing properties on the domain resource that affect server pods (such as `image`, `volumes`, and `env`)
+* Changing fields on the Domain that [affect WebLogic Server instance Pod generation]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md#fields-that-cause-servers-to-be-restarted" >}}) (such as `image`, `volumes`, and `env`)
 * Applying WebLogic Server patches
-* Updating deployed applications for domain home in image
+* Updating deployed applications for Domain in Image
```

Comment:
Updating deployed applications for Domain in Image --> Updating deployed applications for Domain in Image or Model in Image